### ISSUE TITLE
How to play in full screen after recording?

### ISSUE THREAD

**Issue by wenfangdu** [NickeManarin/ScreenToGif#789]
I only found 100% zooming, is it possible to play the gif in full screen?
![image](https://user-images.githubusercontent.com/28700378/105597006-d26df600-5dce-11eb-9bd6-faaa8d5c048b.png)

**Comment by NickeManarin** [comment]
You can reduce the size of the ribbon (top tool bar) by clicking on that arrow to the right, but that's how far you can go within the app. You can try exporting as video or gif and watching the animation on a player too.

**Comment by wenfangdu** [comment]
@NickeManarin Ah, I see what you mean, I was looking for something that does the arrow's job but missed. I guess that proves the bottom right arrow UX is kinda counter-intuitive, I think most Windows users are used to MS word's handling to this case (please ignore the red search icon):

![stage](https://user-images.githubusercontent.com/28700378/105718712-7fc24480-5f5c-11eb-9e74-ae75cae01859.gif)
Could you please consider changing the collapse UX to the above, and perhaps add a full screen mode in a future version?

**Comment by NickeManarin** [comment]
Yes, I'm going to improve the editor window. I already started working on a new ribbon element.
